src/pkg/packaging/snaps/dotnet-sdk-3/dotnet-runtime

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
I believe there are environments where this isn't a great idea but would need to investigate and test. Since this has been the definition from the beginning, I'm inclined to leave it alone for now.